src/main/java/reposense/system/LogsManager.java

```diff
@@ -27,6 +31,7 @@ public class LogsManager {
     // eg. reposense.log.0, in the logs/ folder of the working directory
     private static final String LOG_FOLDER_NAME = "logs";
```

Comment:
I think we should go for a far less generic name like `reposense-logs` or something similar since we're now creating this in the running directory, there's far higher chance of name clash

Reply:
Good idea, it would be bad to conflict with existing files